plugins/copr.py

```diff
@@ -254,6 +269,15 @@ def _remove_repo(cls, repo_filename):
             raise dnf.exceptions.Error(str(e))
 
     @classmethod
+    def _disable_repo(cls, copr_username, copr_projectname):
+        exit_code = call(["dnf", "config-manager", "--set-disabled",
```

Comment:
> We should add this config manager subfunctionality into core lib and call it ftom Python. 

Which functionality? Disable, enable, all? IMHO would be better to provide simple way to call other dnf subcommands from plugin. Even now you can call

    repo = "{}-{}".format(copr_username, copr_projectname)
    cm = self.cli.cli_commands['config-manager'](self.cli)
    cm.configure(['--set-disabled', repo])
    cm.run(['--set-disabled', repo])

it's just not part of API.